app/Database/Repositories/WarningsRepositoryInterface.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * @author    MyBB Group
+ * @version   2.0.0
+ * @package   mybb/core
+ * @license   http://www.mybb.com/licenses/bsd3 BSD-3
+ */
+
+namespace MyBB\Core\Database\Repositories;
+
+interface WarningsRepositoryInterface
+{
+    /**
+     * Get all users.
+     *
+     * @param string $sortBy
+     * @param string $sortDir
+     * @param int $perPage
+     *
+     * @return mixed
+     */
+    //public function allForUser($user_id);
```

Comment:
Is this function going to be implemented, or can it be removed from the interface?